packages/salesforcedx-vscode-core/test/jest/commands/isvdebugging/bootstrapCmd.test.ts

```diff
@@ -11,16 +11,15 @@ describe('isvdebugging unit test', () => {
 
   beforeEach(() => {
     relativeToolsFolderStub = jest.spyOn(projectPaths, 'relativeToolsFolder').mockReturnValue(TOOLS_FOLDER);
-    isvDebugBootstrapExecutorInst = new IsvDebugBootstrapExecutor();
-
   });
 
   it('should be defined', () => {
+    isvDebugBootstrapExecutorInst = new IsvDebugBootstrapExecutor();
```

Comment:
So this wasn't required, but when testing an _instance_ of a class it's always good to create the instance for each test.